addons/web/static/src/legacy/owl_view.js

```diff
@@ -0,0 +1,190 @@
+/** @odoo-module **/
```

Comment:
all the code for owl_view should be moved into account_accountant in enterprise, and treated as an implementation detail, in the period it takes to rewrite the reconciliation widget to owl views